doc/development/tutorials/helloworld.rst

```diff
@@ -115,10 +99,10 @@ the ``text`` parameter.
 This function is a requirement. We use it to plug our new directive into
 Sphinx.
 
-.. code-block:: python
-
-   def setup(app):
-       app.add_directive("helloworld", HelloWorld)
+.. literalinclude:: examples/helloworld.py
+   :language: python
+   :linenos:
+   :lines: 12-
```

Comment:
Yeah. I don't think this will we be too much hassle, in any case. If it would help, I could put a README in the folder saying that people should update the line numbers if they change the objects.

Reply:
Added `README.rst`

Reply:
It seems not added. But I don't think the file will help us...